jina/peapods/runtimes/asyncio/rest/__init__.py

```diff
@@ -58,7 +58,7 @@ async def serve(self, sockets=None):
                 host=self.args.host,
                 port=self.args.port_expose,
                 ws='wsproto',
-                log_level='critical',
+                log_level='info',
```

Comment:
let's make it configurable via `os.environ['JINA_LOG_LEVEL']`, which is the env that globally appreciated by Jina.
in general, we dont want to make our log too verbose. That gives new users a headache

Reply:
`os.environ.get('JINA_LOG_LEVEL', 'critical')`

Reply:
thanks, changed.
I used `error` as the fallback as discussed.